srcpkgs/iptables/patches/0001-Fix-glibc-build.patch

```diff
@@ -0,0 +1,30 @@
+From 8c493cf4aeaa435cbedda574a5620e64d23ddd88 Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens04@gmail.com>
+Date: Wed, 9 Jan 2019 17:11:55 -0600
+Subject: [PATCH 1/2] Fix glibc build
+
+---
+ include/linux/netfilter.h | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git include/linux/netfilter.h include/linux/netfilter.h
+index c3f087a..263d606 100644
+--- include/linux/netfilter.h
++++ include/linux/netfilter.h
+@@ -2,9 +2,12 @@
+ #define __LINUX_NETFILTER_H
+ 
+ #include <linux/types.h>
+-
++#ifndef __GLIBC__
+ #include <linux/in.h>
++#endif
++#ifndef __GLIBC__
```

Comment:
why are there two ifndef's? Just merge into one.

Reply:
Probably happened when I was fixing for each of the C’s. I will take a relook had system issue so having to reinstall

Reply:
https://gitlab.com/ndowens/voidlinux-patches/blob/iptables/update-and-fix-build.patch is updated version